pagopa/io-app
# feat(Bonus Pagamenti Digitali): [#174847263,#174847196] Add network logic to support upsert IBAN

## Short description
Add network logic to support upsert IBAN

- inject `Authentication` and `Ocp-Apim-Subscription-Key` in all bdp requests
- create a custom patch request (io-utils doesn't support code generation for this method)
- mapping response with internal logic values


### how to test
make sure to get the last commit from [io-dev-server](https://github.com/pagopa/io-dev-api-server/pull/28) branch `support-bpd`
